### ISSUE TITLE
Super Monkey Ball Adventure glitches

### ISSUE THREAD

**Issue by zaranrapu** [hrydgard/ppsspp#7674]
(It's machine translation.)
 ppsspp is lacking in the compatibility of this game.
 An analog stick is turned to the front and the speed is increased, but only the speed of about 1/2 will be done than the original speed.
 A certain stage probably becomes impossible to do a goal.

 The speed normally goes up in jpcsp.

 Test play emus.
 ppsspp(x86 and x64) v1.0.1-382 v1.0.1 v0.9.9.1
 jpcsp-x86 v0.7-4c4ff09

Movie link.
http://l2.upup.be/f/bcbKJRUuEm.3gp?nowtime=1428758847

**Comment by daniel229** [comment]
Change CPU clock to 46 helps it

**Comment by zaranrapu** [comment]
Thank you very much.
But this should be corrected by default.
I don't usually notice.

**Comment by ppmeis** [comment]
Confirmed working using CPU clock set to 46.

**Comment by unknownbrackets** [comment]
Is there anything useful in the logs around that time?

-[Unknown]

**Comment by daniel229** [comment]
here are the debug logs
rename jpg to 7z
![cpu 46](https://cloud.githubusercontent.com/assets/3481559/11916658/1d90b5ca-a71b-11e5-90f4-6ff0eac54c73.jpg)
![default cpu](https://cloud.githubusercontent.com/assets/3481559/11916659/1d91a232-a71b-11e5-86f4-a53813bef45f.jpg)

**Comment by daniel229** [comment]
That Prevent FPS from exceeding 60 also big improves the game speed like god of war.
unchecked that option,the monkey running at a even slower speed below 1/4 speed.

**Comment by unknownbrackets** [comment]
We've removed that setting now, so this might've gotten worse.  But other fixes could've improved it too.

We probably have some sort of timing issue affecting this.

-[Unknown]

**Comment by Panderner** [comment]
@unknownbrackets can you add this game force 60 FPS via compat.ini?

**Comment by Panderner** [comment]
This Game running slow. but It can approving the performance by underclocking the CPU

**Comment by hrydgard** [comment]
I added it, how does it work now?

**Comment by Panderner** [comment]
> I added it, how does it work now?

Yes Performance improved running full speed for mid-range devices in high resolutions, but low end devices struggles any 3D games runs at 60 fps (except Dragon Ball Z Shin Budokai)

**Comment by XiaomiRedmi9** [comment]
How about this now?
